src/Pokemon.java

```diff
@@ -0,0 +1,83 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.DoubleToIntFunction;
+
+public class Pokemon {
+    // deze variabelen meerdere keren voor
+    private final String type; /*""*/
+    private final String name;/*""*/
+    int level;
+    int hp;
+    private final String food;/*""*/
+    private final String sound;/*""*/
+    List<Pokemon> pokemons;
```

Comment:
Een list<Pokemon> en objecten van type Pokemon in de Pokemon class zelf, is vrij bijzonder. Deze passen beter in de Pokemontrainer class.